gurubu-client/src/app/api/og/route.tsx

```diff
@@ -0,0 +1,55 @@
+              fill="black"
+            />
+          </svg>
```

Comment:
this svg is pretty big, let's import it from somewhere

Reply:
I exported the logo with text as png format, I removed this whole messy svg data :).